src/ui/GenioWindow.cpp

```diff
@@ -3612,6 +3668,14 @@ GenioWindow::_UpdateProjectActivation(bool active)
 	fToolBar->SetActionEnabled(MSG_BUILD_PROJECT, active);
 	
 	if (active == true) {
+		fBuildItem->SetEnabled(true);
```

Comment:
@Freaxed Are these lines (3671-3676 and 3712) correct? I think they were removed by a previous commit

Reply:
ok, this was my understanding too but I wanted to double check. I'll remove them